docs/getting-started/local-setup.md

```
@@ -0,0 +1,106 @@
+# Local Development Setup
+
+Open Chat Studio uses [UV](https://docs.astral.sh/uv/getting-started/installation/) and [Invoke](https://www.pyinvoke.org/) for dev automation.
+
+### Prerequisites
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)